plugins/out_azure/azure_conf.c

```diff
@@ -110,6 +115,15 @@ struct flb_azure *flb_azure_conf_create(struct flb_output_instance *ins,
         return NULL;
     }
 
+    /* config: 'time_generated' */
+    tmp = flb_output_get_property("time_generated", ins);
+    if (tmp) {
+        ctx->time_generated = bool_value(tmp);
```

Comment:
We support `flb_utils_bool` function.
Could you use the API ?

Reply:
Ofcourse! I wasn't aware of that API. I've digged through the _influxdb.c_ sourcecode and found it there. Looks like it already got fixed in influx as well. https://github.com/fluent/fluent-bit/commit/11323419cc060fbdf4546903fc52289a1343a58f#diff-3b3a56855868026d54acd7df3827545b9e3ecc04c4cc8b44239289e4c99d550d

Reply:
Thank you.